### DISCUSSION TITLE
SSG and SSR errors while deploying to `zeit/now`

### DISCUSSION THREAD

**Discussion by mgranados** [vercel/next.js#10988] [Help]
```
Module not found: Can't resolve 'dns' in '/zeit/fa3e271/node_modules/mongodb/lib'
00:11:08.674
> Build error occurred
00:11:08.675
Error: > Build failed because of webpack errors
00:11:08.675
```

Getting now that error on the deployment to now that seems related to executing the db function call on the `getStaticProps` function of the component. Any leads on how to deploy now? Works perfectly on local with `next` on local

**Comment by Timer** [comment]
Could you please provide a reproducible demo? This is currently unactionable with the information provided.
